Neos.Flow/Classes/Security/TransientAccount.php

```diff
@@ -0,0 +1,124 @@
+<?php
+namespace Neos\Flow\Security;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Security\Authentication\AuthenticationProviderName;
+use Neos\Flow\Security\Authentication\CredentialsSource;
+use Neos\Flow\Security\Policy\Role;
+use Neos\Flow\Security\Policy\Roles;
+
+/**
+ * An account that is not persisted in the database
+ *
+ * This can be used for scenarios where no corresponding entity is stored in the local database, for
+ * example when using a Single Sign-On service.
+ *
+ * @Flow\Proxy(false)
+ * @api
+ */
+final class TransientAccount implements AccountInterface
```

Comment:
@bwaidelich We can/should use this inside the FileBasedSimplekeyProvider, where we create a Account object - that will create fine example for, how to use it :-D

Reply:
Yes, good point, added it with 2e2a7e07a6475ce49d492af4978cee12b5d25ed8 (not tested yet)